scripts/launch/connect-to-cluster/default.nix

```diff
@@ -64,13 +64,6 @@ in pkgs.writeScript "${executable}-connect-to-${environment}" ''
   mkdir -p ${stateDir}/logs
 
   echo "Launching a node connected to '${environment}' ..."
```

Comment:
I think it should be install time for daedalus, but when ran directly with nix it should be run time as it currently is. Mainly because anything nix installs goes into a world readable store. In general in the nix world, this would happen in a wrapper script before the application is started like we're doing for exchanges with connect-to-cluster or as a preStart script for a systemd service.

Reply:
Since there's no installation of cardano-node so to speak, I do agree indeed. I believe that's also aligned with what we discussed, we can bundle the certificate generation tool alongside cardano-node with nix, and call it as part of the `connect-to-cluster` upon starting it :+1: 

For daedalus, it should pretty much replace the call to `build-certificate-*` which is part of the post-install hook.

Reply:
i'm thinking it should just be linked into `cardano-launcher` at runtime, along with a few more checks to re-run it if any vital files are missing